challenge_opener/src/androidTest/java/com/example/challengeopener/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.challengeopener
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
+    @Test
+    fun useAppContext() {
+        // Context of the app under test.
+        val appContext = InstrumentationRegistry.getInstrumentation().targetContext
+        assertEquals("com.example.challengeopener", appContext.packageName)
+    }
+}
```

Comment:
Please, add a new line at the end of the line. (See https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b)